reader/.flowconfig

```diff
@@ -8,5 +8,6 @@
 [libs]
 
 [options]
+server.max_workers=1
```

Comment:
Why set the max_workers to 1? Wouldn't allowing more workers speed up the flow checks?

Reply:
Flow appears to perform "parallelism" by spawning new parsing threads on an interval. CircleCI broke on the flow parsing step after x amount of these workers were spawned and exhausted the memory. After trial and error this was the only thing that fixed it.